### PR TITLE
Administrator's Guide updates

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
Just a few small adjustments to this guide

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/administrators_guide/build.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ further_reading:
 
 After you plan your Datadog installation design and best practices, concentrate on the construction of Datadog itself, understanding what needs to be installed, and the best way to achieve that.
 
-As you scale your IT footprint, it's important to develop standards, methods, and practices for how software is installed and used. Engineering effort is dedicated to developing precise steps for installing and configuring software in a repeatable, standardized manner, while still allowing for a significant level of flexibility. Without these standards, reliability can be impacted. In this section, we describe how Datadog can most efficiently interact with those standards, methods, and practices.
+As your IT footprint grows, establish standards for software installation and usage. Develop precise, repeatable steps for configuring software while maintaining flexibility. This ensures reliability. This section explains how Datadog can efficiently integrate with these standards.
```
Comment (janine-c) [inline content/en/administrators_guide/build.md:12]: I'm loving the improved conciseness here, but I was having some trouble following the relationships between these sentences. I'm wondering if we can reinstate some of the transitional language between sentences, maybe like:

```suggestion
As your IT footprint grows, you need to establish standards for software installation and usage. To do this, it's important to develop precise, repeatable steps for reliably configuring software while maintaining the flexibility you need. This section explains how Datadog can efficiently integrate with these standards.
```

**Review by janine-c — CHANGES_REQUESTED** [review]
Looks fantastic, Alicia! I have one comment that I'm happy to discuss further if you like 🙂

File: content/en/administrators_guide/build.md (hunk 1 above)
Comment (aliciascott) [reply content/en/administrators_guide/build.md:12]: that is much better thank you!

**Review by janine-c — APPROVED** [review]

**Pull request merged**